config/config.exs

```diff
@@ -0,0 +1,4 @@
+use Mix.Config
+
+config :ex_unit,
+  assert_receive_timeout: 1000
```

Comment:
What test is failing?
Having 1000 we're just hiding the bad test, we should rather to rewrite it.